modules/routes/Forums/forum_route.py

```diff
@@ -0,0 +1,91 @@
+from flask import request, jsonify
+from app import app
+from modules.models.account import AccountType
+from modules.routes.Forums.forum_schema import (
+    ForumSchema,
+    DiscussionThreadSchema,
+    DiscussionReplySchema,
+    NewDiscussionThreadSchema,
+    NewDiscussionReplySchema,
+
+)
+from modules.utils.db import db
+from modules.utils.route_utils import authenticate, fetch_session, protected_route
+
+@app.route("/forums/course/<string:course_code>", methods=["GET"])
```

Comment:
Where is the route protection??

```suggestion
@app.route("/forums/course/<string:course_code>", methods=["GET", "POST"])
@protected_route()
```